src/candidates/sign/store.js

```diff
@@ -19,6 +21,9 @@ const mutations = {
   },
   [SET_CANDIDATE_EMAIL] (state, email) {
     state.email = email;
+  },
+  [SET_SUCCESS_REGISTRATION] () {
+    state.successRegistration = true;
```

Comment:
Do I understand correctly that the store is always reset on page refresh? I was afraid `successRegistration` would persist with the `true` value, but it appears not to. I was able to sign up, manually navigate to `sign/success-registration` and it would properly redirect me to the Dashboard page.

Reply:
@lilsweetcaligula Yes, state will be wiped on page refresh. But in your experiment I think it's the `beforeEnter` in `/src/candidates/sign/router.js` that does the trick. :)
Basically, after you are logged in (have a token in local storage), you will be redirected to dashboard from each sign route.

Reply:
@tetiana-kolmychek-devv Do you think `state.successRegistration` should be explicitly reset to `false` somewhere in the code?

Like is it possible to somehow "trick" the app state here into getting stuck at `successRegistration === true`, even if, say, a user logs out and never tries to sign back in?

Reply:
@lilsweetcaligula if you want to make sure, that `successRegistration` is false after user leaves `success-registration` route, you can always change state in component `SignMessage` in `destroyed` function. It will be triggered when component instance will be destroyed.
You can see how it's applied in `Logout` component, for example.

If you want to make such a change, please think about user-experience with routes and redirects in case user presses `back` in browser after being redirected to login code page from success registration.

Reply:
@tetiana-kolmychek-devv Thanks, I'll check it out.